.eslintrc.js

```diff
@@ -244,6 +244,12 @@ module.exports = {
   plugins: ['prettier'],
   extends: ['plugin:prettier/recommended'],
   overrides: [
+    {
+      files: ['esm/**/*.js'],
```

Comment:
why don't we need the same thing for esm?

Reply:
pretty sure we don't need this at all.

Reply:
You're right, that was left over from an earlier iteration